src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IndexerAccess.cs

```diff
@@ -105,7 +115,7 @@ private BoundExpression VisitIndexerAccess(BoundIndexerAccess node, bool isLeftO
                 // This node will be rewritten with MakePropertyAssignment when rewriting the enclosing BoundAssignmentOperator.
 
                 return oldNodeOpt != null ?
-                    oldNodeOpt.Update(rewrittenReceiver, indexer, rewrittenArguments, argumentNamesOpt, argumentRefKindsOpt, expanded, argsToParamsOpt, null, isLeftOfAssignment, type) :
+                    oldNodeOpt.Update(rewrittenReceiver, indexer, rewrittenArguments, argumentNamesOpt, argumentRefKindsOpt, expanded, argsToParamsOpt, oldNodeOpt.BinderOpt, isLeftOfAssignment, type) :
```

Comment:
>oldNodeOpt.BinderOpt [](start = 152, length = 20)

Can we revert this change?

Reply:
> We can... but doesn't the old behavior look wrong to you?

I think the binder there is for IOperation factory, which is supposed to deal with nodes produced by initial binding. 

---
In reply to: [216492768](https://github.com/dotnet/roslyn/pull/29591#discussion_r216492768) [](ancestors = 216492768)

Reply:
Ah, I see. I'll revert.